src/Type/WPEnumType.php

```diff
@@ -96,6 +97,17 @@ private static function prepare_values( $values, $type_name ) {
 		$values = apply_filters( 'graphql_' . lcfirst( $type_name ) . '_values', $values, $type_name );
 		$values = apply_filters( 'graphql_' . $type_name . '_values', $values, $type_name );
 
+		// map over the values and if the description is a callable, call it
+		$values = array_map(
+			static function ( $value ) {
+				if ( ! is_array( $value ) ) {
+					$value = [ 'value' => $value ];
+				}
+				return TypeRegistry::prepare_config_for_introspection( $value );
```

Comment:
I hate that we're doing this here, and that `prepare_config_for_introspection()` is public), but I don't have a better solution since we don't have an Abstract type, and utils are even worse.

Just sharing in care you have a better idea, if not just hit resolve.